app/src/main/java/pl/skipcode/basekotlinapp/feature/main/navigation/MainRouter.kt

```diff
@@ -20,4 +23,8 @@ class MainRouter(
         )
         finish()
     }
+
+    override fun goToAboutFragment() {
+        activity.pushFragment(AboutFragment(), BaseActivity.ANIM.FADE_IN)
```

Comment:
w jednej linijce bym to robił `override fun goToAboutFragment() = activity.pushFragment(....)`